src/shared/constants.ts

```diff
@@ -67,8 +67,11 @@ export enum QueryRefreshRates {
 
 export const DEFAULT_LIST_LIMIT = 50;
 
-export const TRANSAK_API_KEY_PRODUCTION = '7300ebf7-c657-46b1-9c72-c0d91bbed0a8';
+// TODO: Change to prod app id
+// export const COINBASE_APP_ID_PRODUCTION = '';
+export const COINBASE_APP_ID_STAGING = 'ca72edbc-0a08-468c-952d-6cde50b1dce6';
```

Comment:
Ideally config values should be set by the environment, not commited in code https://12factor.net/config

Reply:
Yeah this would be good. We can leave as is for now, but a separate task could be to add .env (and move over existing webpack env config)

Reply:
I did add .env support in the api key header PR that is still open. I will create an issue to come back to this after that is merged.